pkmodel/model.py

```diff
@@ -4,14 +4,78 @@
 
 class Model:
     """A Pharmokinetic (PK) model
+    -------
+    INPUTS: dictionary describing the model
+    The dictionary should have the following template:
+    {
+        'name': name <str>,
+        'CL': CL <float> [mL/h],
+        'Vc': Vc <float>, [mL]
+        'ka': ka <float>, [1/h] (if subcutaneous)
+        'Qp1': Qp1 <float>, [mL/h] (if one peripheral used)
+        'Vp1': Vp1 <float>, [mL] (if one peripheral used)
+        'Qp2': Qp1 <float>, [mL/h] (if two peripheral used)
+        'Vp2': Vp1 <float>, [mL] (if two peripheral used)
+        'Qp3': Qp1 <float>, [mL/h] (if three peripheral used)
+        'Vp3': Vp1 <float>, [mL] (if three peripheral used)
+        ... etc
+    }
 
-    Parameters
-    ----------
+    !! The peripheral compartiments must be added starting from
+    1 and increasing in increments of 1 up to the maximum number
+    peripheral compartiments used !!
 
-    value: numeric, optional
-        an example paramter
+    !! At a minimum the model must have a main compartiemnt with
+    CL and Vc given and a name !!
+    ----------
+    OUTPUTS: model object that holds all the properties as attributes
```

Comment:
move the 'model object that ...' underneath the underline on outputs - again to make the docs look nicer